src/MSBuild/XMake.cs

```diff
@@ -795,7 +801,13 @@ string commandLine
                 // Wait for any pending cancel, so that we get any remaining messages
                 s_cancelComplete.WaitOne();
 
-                MSBuildEventSource.Log.MSBuildExeStop();
+#if FEATURE_GET_COMMANDLINE
+                MSBuildEventSource.Log.MSBuildExeStop(commandLine);
+#else
+                if (MSBuildEventSource.Log.IsEnabled()) {
+                    MSBuildEventSource.Log.MSBuildExeStop(commandLine.ToString());
```

Comment:
Is `ToString()` sufficient here? Does it not need to be `string.Join(" ", commandLine)` or similar?

Reply:
I think both would work, but ToString() will have more punctuation and be harder to read.